lib/workos/client.rb

```diff
@@ -81,8 +81,13 @@ def user_agent
       ].join('; ')
     end
 
+    def extract_error(errors)
```

Comment:
Let's throw this at the bottom of the file as a private method.

```rb
def public_method
end

private

def extract_error(errors)
end
```

Reply:
- [ ] move method to bottom and make private